.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,12 +1,15 @@
 # Title
 
-<!-- Thanks for sending a pull request! Make sure to follow the contributing guidelines. -->
+<!-- Thanks for sending a pull request! Make sure to follow the contributing guidelines: https://github.com/FreeTubeApp/FreeTube/blob/development/CONTRIBUTING.md -->
 <!-- Important note, we may remove your pull request if you do not use this provided PR template correctly. -->
 
 ## Pull Request Type
 <!-- Please select what type of pull request this is: [x] -->
+<!-- Please remove what is not applicable! -->
 - [ ] Bugfix
 - [ ] Feature Implementation
+- [ ] New Build
```

Comment:
Also confused about this

Reply:
So i added this because i saw this header in our last release notes https://github.com/FreeTubeApp/FreeTube/releases/tag/v0.18.0-beta

Reply:
That rarely happens, no need a dedicated option for it IMO

Reply:
You're right. I looked at previous FT release notes to confirm this.